File: RiLiXuanZeQi/app/src/main/java/com/zhifei/test/rilixuanzeqi/MainActivity.java
package com.zhifei.test.rilixuanzeqi;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.DatePicker;
import android.widget.TimePicker;

import java.util.Calendar;

public class MainActivity extends AppCompatActivity {

    private DatePicker datePicker;
    private TimePicker timePicker;
    private Calendar cal;
    private int year;
    private int month;
    private int day;
    private int hour;
    private int minute;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        datePicker = (DatePicker) findViewById(R.id.datePicker);
        timePicker = (TimePicker) findViewById(R.id.timePicker);
        //获取日历一个对象
        cal = Calendar.getInstance();
        //获取年月日时分秒的信息

        year = cal.get(Calendar.YEAR);
        month = cal.get(Calendar.MONTH)+1;//获取当前月份+1处理



    }
}

File: GsyPlayer/app/src/main/java/baiye/ali/com/gsyplayer/MainActivity.kt
package baiye.ali.com.gsyplayer

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // gsyVideoPlayer.setUp("http://ali.cdn.kaiyanapp.com/1497859203791_c9c1a602.mp4?auth_key=1514448702-0-0-487ed3fcb2cea795bd9bdf840f70dba0",false,"没有");
        //StandardGSYVideoPlayer standardGSYVideoPlayer = new StandardGSYVideoPlayer(this);
        // GSYVideoPlayer gsyVideoPlayer =  findViewById(R.id.gsyplay);
        //gsyVideoPlayer.setUp("http://ali.cdn.kaiyanapp.com/1497859203791_c9c1a602.mp4?auth_key=1514448702-0-0-487ed3fcb2cea795bd9bdf840f70dba0",false,"没有");

        gsy_player!!.setUp("http://ali.cdn.kaiyanapp.com/1497859203791_c9c1a602.mp4?auth_key=1514448702-0-0-487ed3fcb2cea795bd9bdf840f70dba0",false,"没有")

    }
}
